js/about/preferences.js

```diff
@@ -1128,4 +1233,12 @@ class AboutPreferences extends React.Component {
   }
 }
 
+let formattedDateFromTimestamp = function (timestamp) {
+  var date = new Date(timestamp)
+  var dateStr = date.toDateString()
+  var [month, day, year] = dateStr.split(' ').slice(1)
+  dateStr = month + ' ' + day + ', ' + year
+  return dateStr
```

Comment:
this may be hard to internationalize; i would rather just return `date.toLocaleDateString()` here